packages/logs/src/domain/logger.ts

```diff
@@ -40,19 +40,19 @@ export class Logger {
 
   @monitored
   log(message: string, messageContext?: object, status: StatusType = StatusType.info) {
-    this.handleLogStrategy({ message, context: deepClone(messageContext) as Context, status }, this)
+    this.handleLogStrategy({ message, context: sanitize(messageContext) as Context, status }, this)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏would it worth to sanitize `message` as well?